### DISCUSSION TITLE
SVG Units

### DISCUSSION THREAD

**Discussion by lythhh** [cduck/drawSvg#71] [Q&A] [answered]
hello!
Class Drawing in module drawSvg.drawing is 'Drawing(width, height, origin=(0, 0), idPrefix='d', displayInline=True, **svgArgs)'
i want to konw the unit of  width or height in Drawing() . Is it the pt?how can i use the unit of 'mm'?Thank you very much!

**Comment by cduck** [comment, marked as answer]
SVGs are "scalable" so the units can mean whatever you like.  The printed or displayed length of the drawing unit is determined by how the graphic is printed or displayed.

[Read this](https://www.w3.org/TR/SVG/coords.html#EstablishingANewSVGViewport) for more background.  Note: drawSvg does not currently support units in width and height as described in that document.  You could overwrite
```python
d = draw.Drawing(...)
d.width = "100mm"
d.height = ...
d.viewport = ...
```
for example but this is not supported and may change or break in future versions.
